Tests/PointFreeTests/UpdateProfileTests.swift

```diff
@@ -12,6 +12,7 @@ import Optics
 
 class UpdateProfileTests: TestCase {
   func testUpdateNameAndEmail() {
+    #if !os(Linux)
```

Comment:
had to omit these tests from linux since they write to the flash which is stored in the sessions